### PR TITLE
Add KSN Open Information

### PR DESCRIPTION
**Pull request by prabowo02**
Before merging, ensure the following:
- The KSN Open Contest has been created in TLX (for all three days)
- Peraturan Kompetisi has been added

### CODE REVIEW

File: terbuka.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<br>
+
+### Jadwal dan Teknis Kontes
+
+- Secara umum, teknis sesuai dengan yang tercantum pada halaman [Peraturan Kompetisi](peraturan.html).
+- Kontes dibuka dalam rentang waktu sebagai berikut.
+  * **Hari 0**: 12 Oktober 2020 16:05 WIB - 13 Oktober 2020 05:05 WIB
+    (sesi uji coba, **tidak** mempengaruhi nilai akhir)
+  * **Hari 1**: 13 Oktober 2020 13:35 WIB - 14 Oktober 2019 05:05 WIB
+  * **Hari 2**: 14 Oktober 2020 13:35 WIB - 15 Oktober 2019 05:05 WIB
```
Comment (yonasadiel) [inline terbuka.md]: the year is not updated yet

File: open.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+### Schedule and Technical Details
+
+- The contests will be held at the following time ranges:
+* **Day 0**: [12 October 2020 09:05 - 22:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+KSN+Open+Contest+2020+Day+0&iso=20201012T1605&p1=108&ah=13&am=00) (trial session)
+* **Day 1**: [13 October 2020 06:35 - 22:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+KSN+Open+Contest+2020+Day+1&iso=20201013T1335&p1=108&ah=15&am=30)
+* **Day 2**: [14 October 2020 06:35 - 22:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+KSN+Open+Contest+2020+Day+2&iso=20201014T1335&p1=108&ah=15&am=30)
+
+- Each contestant can select **any 5-hour window** within the time range to do the contest for each competition day.
+- All three contests are now available on [TLX](https://tlx.toki.id/contests). Please register in those contests.
+- Each contestant may start the contest any time within the time range, by clicking the timer button.
```
Comment (yonasadiel) [inline open.md]: Maybe you want to clarify the "timer" button. I think its currently "REGISTER" ?

**Comment by prabowo02** [comment]
Since Faishol is not the Open KSN coordinator, TODO: update the contact to either my email or yonasadiel's email

**Comment by prabowo02** [comment]
Above todo list has been done, and tested locally. Can be merged

**Review by fushar — APPROVED** [review]

**Pull request merged**